Dispatcher/Controllers/Homepage/HomeViewController.swift

```diff
@@ -23,11 +26,43 @@ class HomeViewController: UIViewController {
         articlesTableView.dataSource = self
```

Comment:
We can place all of this tableview-related-stuff inside a dedicated function like "setupTableView"